src/js/views/patients/sidebar/action/action-sidebar-attachments-views.js

```diff
@@ -0,0 +1,36 @@
+import hbs from 'handlebars-inline-precompile';
+import { View, CollectionView } from 'marionette';
+
+import { alphaSort } from 'js/utils/sorting';
+
+import 'scss/modules/sidebar.scss';
+
+import './action-sidebar.scss';
+
+const AttachmentView = View.extend({
+  className: 'u-margin--t-16',
+  template: hbs`
+    <a class="action-sidebar__attachment-filename" target="_blank" href={{path}}>{{filename}}</a>
+    <div>
+      <a class="action-sidebar__attachment-download" href={{path}} download>
+        {{far "download"}} <span>Download</span>
+      </a>
+    </div>
+  `,
+  templateContext() {
+    return {
+      filename: this.model.get('path').split('/').pop(),
+    };
+  },
+});
+
+const AttachmentsView = CollectionView.extend({
+  childView: AttachmentView,
+  viewComparator(viewA, viewB) {
```

Comment:
If there are more than one attachments, do we want them sorted in a certain way?

Could be by `created_by` or `updated_at`.

Reply:
Looks like `viewComparator: 'created_at'` is ascending. And we want descending (newest to oldest).